data-query/src/main/java/gov/va/api/health/dataquery/service/controller/organization/R4OrganizationTransformer.java

```
@@ -55,15 +59,55 @@
             .build());
   }
 
-  static List<Identifier> identifier(Optional<String> maybeNpi) {
-    if (isBlank(maybeNpi)) {
+  static List<Identifier> identifiers(
+      Optional<String> maybeNpi, Optional<FacilityId> maybeFacility) {
+    List<Identifier> results = new ArrayList<>(2);
+    if (!isBlank(maybeNpi)) {
+      results.add(
+          Identifier.builder()
+              .system("http://hl7.org/fhir/sid/us-npi")
+              .value(maybeNpi.get())
+              .build());
+    }
+    if (maybeFacility.isPresent()) {
+      String facilityIdPref;
+      switch (maybeFacility.get().type()) {
+        case HEALTH:
+          facilityIdPref = "vha_";
+          break;
+        case BENEFITS:
+          facilityIdPref = "vba_";
+          break;
+        case VET_CENTER:
+          facilityIdPref = "vc_";
+          break;
+        case CEMETERY:
+          facilityIdPref = "nca_";
+          break;
+        default:
+          facilityIdPref = "ncas_";
```

This is incorrect. Per the wiki page:
```
nca with s for non-national (i.e. non-VA) cemeteries, e.g., ncas_1051
```
That prefix should match with the datamart enum, and the default should throw an `InvalidStateException`